MediaBrowser.Model/IO/AsyncFile.cs

```diff
@@ -26,6 +26,14 @@ public static class AsyncFile
             Options = FileOptions.Asynchronous
         };
 
+        /// <summary>
+        /// Creates, or truncates and overwrites, a file in the specified path.
+        /// </summary>
+        /// <param name="path">The path and name of the file to create.</param>
+        /// <returns>A <see cref="FileStream" /> that provides read/write access to the file specified in path.</returns>
+        public static FileStream Create(string path)
```

Comment:
```suggestion
        public static FileStream CreateOpen(string path)
```
just an idea to signal that it returns a filestream?

Reply:
This class is intended to mirror the System.IO.File class but with async filestreams by default.

https://learn.microsoft.com/en-us/dotnet/api/system.io.file.create?view=netframework-4.7#system-io-file-create(system-string)